src/components/Layout/index.tsx

```diff
@@ -46,6 +46,12 @@ const Layout: FC<Props> = ({ children }) => {
                         router.pathname === '/mine'
                     )}
                     |
+                    {navBarContent(
+                        'Standup',
+                        '/standup',
+                        router.pathname === '/standup'
+                    )}
+                    |
                     {navBarContent(
                         'DS',
                         '/challenges',
```

Comment:
Nit:
Hello @Pratiyushkumar, If you pull latest changes from develop...these /challenges route is comment
https://github.com/Real-Dev-Squad/website-status/blob/develop/src/components/Layout/index.tsx#L48